### PR TITLE
fixing python lint tests

### PR DESCRIPTION
**Pull request by lev-titov**
image

### CODE REVIEW

File: packages/robot/odahuflow/robot/libraries/model.py [hunk 1]
```diff
@@ -44,21 +44,21 @@ def get_model_info(edge, token, model_name, model_version=None):
         :param token: model API JWT token
         :return: dict -- response
         """
-        headers = {"Authorization": "Bearer {}".format(token)}
+        headers = {"Authorization": f"Bearer {token}".format()}
```
Comment (BPylypenko) [inline packages/robot/odahuflow/robot/libraries/model.py]: I suppose that there is no need for it)

```suggestion
        headers = {"Authorization": f"Bearer {token}"}
```

File: packages/sdk/odahuflow/sdk/clients/api_aggregated.py [hunk 2]
```diff
@@ -269,7 +269,7 @@ def parse_resources_file_with_one_item(path: str) -> OdahuflowCloudResourceUpdat
     """
     resources = parse_resources_file(path)
     if len(resources.changes) != 1:
-        raise Exception('{!r} should contain 1 item, but {!r} found'.format(path, len(resources)))
+        raise Exception(f'{path!r} should contain 1 item, but {len(resources)} found')
```
Comment (BPylypenko) [inline packages/sdk/odahuflow/sdk/clients/api_aggregated.py:272]:
```suggestion
        raise Exception(f'{path!r} should contain 1 item, but {len(resources)!r} found')
```
Or it doesn't make a change?

File: packages/sdk/odahuflow/sdk/io_proc_utils.py [hunk 3]
```diff
@@ -40,14 +40,13 @@ def run(*args: str, cwd=None, stream_output: bool = True):
...
         return exit_code
     else:
         with subprocess.Popen(args, env=cmd_env, stdout=subprocess.PIPE, stdin=subprocess.PIPE,
                               stderr=subprocess.PIPE, cwd=cwd, universal_newlines=True) as child:
             stdout, stderr = child.communicate()
             exit_code = child.wait()
         if exit_code != 0:
-            raise Exception("Non-zero exit code: %s\n\nSTDOUT:\n%s\n\nSTDERR:%s\n ======" %
-                            (exit_code, stdout, stderr))
+            raise Exception(f'Non-zero exit code: {exit_code}\n\nSTDOUT:\n{stdout}\n\nSTDERR:{stderr}\n ======')
```
Comment (BPylypenko) [inline packages/sdk/odahuflow/sdk/io_proc_utils.py]:
```suggestion
            raise Exception(f'Non-zero exit code: {exit_code}\n\n'
                f'STDOUT:\n{stdout}\n\n'
                f'STDERR:{stderr}\n ======')
```

File: packages/sdk/odahuflow/sdk/gppi/executor.py [hunk 4]
```diff
@@ -231,8 +231,7 @@ def execute(self, command: str, cwd: str = None, stream_output: bool = True):
             _logger.info(f'use_current_env flag = False. Start to execute command in {self.exec_env}')
             exit_code, output, err_ = self.exec_env.execute(command, cwd=cwd, stream_output=stream_output)
 
-        _logger.info('Subprocess result: %s\n\nSTDOUT:\n%s\n\nSTDERR:%s\n ======' %
-                     (exit_code, output, err_))
+        _logger.info(f'Subprocess result: {exit_code}\n\nSTDOUT:\n{output}\n\nSTDERR:{err_}\n ======')
```
Comment (BPylypenko) [inline packages/sdk/odahuflow/sdk/gppi/executor.py]:
```suggestion
        _logger.info(f'Subprocess result: {exit_code}\n\n'
                    f'STDOUT:\n{output}\n\n'
                    f'STDERR:{err_}\n ======')
```

**Review by BPylypenko — CHANGES_REQUESTED** [review]
Please, make changes (1 requered, 3 proposals).

File: packages/sdk/odahuflow/sdk/clients/api_aggregated.py (hunk 2 above)
Comment (lev-titov) [reply packages/sdk/odahuflow/sdk/clients/api_aggregated.py:272]: len returns an int, so yeah, there is no point in repr

**Review by BPylypenko — APPROVED** [review]

**Review by karbyshevd — APPROVED** [review]

**Review by karbyshevds — APPROVED** [review]

**Pull request merged**